.github/ISSUE_TEMPLATE/test-plan.md

```diff
@@ -0,0 +1,62 @@
+## Test plan
+
+## Feature
+
+Name:
+
+Description:
+
+Group: CX / Infra / MDM / Website
+
+## DRIs
+
+Engineer:
+
+QA: 
+
+## Changes
+
+Breaking Changes: Y/N
+
+Clean database: Y/N
+
+## Risk assessment
+
+Risk level: Low / Medium / High
+
+Risk area: 
+
+## Testing requirements
+
+- Browser/OS compatibilty: Y/N
+- Load testing: Y/N
+- Regression: Y/N
+
+#### Automated:
+
+- Fleet: Covered / Will not cover / Scoped
+- QAWolf: Covered / Will not cover / Scoped
+
+## Manual testing steps
+
+Manual:
+1. Step 1
+2. Step 2
+3. Step 3
+
+## Testing notes
+Any additional testing notes relevant to this story.
+
+## Tools
+Any additional tools required (local tuf service, api testing (postman), specific software)
+
+## Results
+
+1. [ ] Engineer: Add comment to testing plan confirming you successfully completed the testing plan. 
+2. [ ] QA: Add comment to testing plan confirming you successfully completed the testing plan. 
+
+## Results notes
+
+Engineer: 
+
+QA: 
```

Comment:
Github comments on the original user story could work and keep surface area small (how I've done it in the past).